tests/cypress/integration/actions_tasks3/case_109_dummy_cloud_storage.js

```diff
@@ -0,0 +1,196 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+const {
+    cloudStoragesDummyDataGoogleStorage,
+    cloudStoragesDummyDataAzureContainer,
+    cloudStoragesDummyDataAWSBucket,
+} = require('../../support/dummy-data');
+
+context('Dummy Cloud storages.', { browser: '!firefox' }, () => {
+    const caseId = '109';
+    const imageFolder = '../integration/actions_tasks3/assets/case_109';
+
+    function testCreateDummyStorage(dummyCS) {
```

Comment:
Why does the function have such a name? It isn't corresponding to content.

Reply:
Replace with ```testCreateDummyStorageItem```. It also creates a fake storage item.